### DISCUSSION TITLE
Why is `\d Post` is returning `Did not find any relation named "Post"` while I can clearly see all the relations by running `\d`?

### DISCUSSION THREAD

**Discussion by MaxHass12** [prisma/prisma#25289] [Q&A]
### Question

I am trying to get started with Prisma.

After running `npx prisma migrate dev --name init` it is showing the success message.

But I am unable to see the contents of the created tables through SQL queries.

`\d` clearly shows that the table exists buts `\d <table_name>` or `SELECT * FROM <table_name>;` returns `Did not find any relation`.

I think I am making some obvious mistake. Please give me a direction. Thanks

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Post {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  title     String   @db.VarChar(255)
  content   String?
  published Boolean  @default(false)
  author    User     @relation(fields: [authorId], references: [id])
  authorId  Int
}

model Profile {
  id     Int     @id @default(autoincrement())
  bio    String?
  user   User    @relation(fields: [userId], references: [id])
  userId Int     @unique
}

model User {
  id      Int      @id @default(autoincrement())
  email   String   @unique
  name    String?
  posts   Post[]
  profile Profile?
}
```

```
postgres=# \d
                 List of relations
 Schema |        Name        |   Type   |  Owner   
--------+--------------------+----------+----------
 public | Post               | table    | postgres
 public | Post_id_seq        | sequence | postgres
 public | Profile            | table    | postgres
 public | Profile_id_seq     | sequence | postgres
 public | User               | table    | postgres
 public | User_id_seq        | sequence | postgres
 public | _prisma_migrations | table    | postgres
(7 rows)

postgres=# \d Post
Did not find any relation named "Post".

```

- OS: MacOS
- Database: PostgreSQL
- Node.js version: v21.7.1

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @MaxHass12 👋 

Thank you for raising this question.

To display the `Post` Table in psql correctly, you need to use double quotes on the Post. 

i.e: `\d "Post";`

Also, to retrieve the `Post` record, you can do:
```sql
SELECT * FROM "Post";
```
With this, you should be able to get things working correctly.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
